subcommands/waves/rollout.go

```diff
@@ -10,25 +15,81 @@ import (
 )
 
 func init() {
-	cmd.AddCommand(&cobra.Command{
-		Use:   "rollout <wave> <group>",
-		Short: "Rollout a given wave to devices in a given device group",
-		Long: `Rollout a given wave to devices in a given device group.
-Upon rollout a wave becomes available as an update source for production devices in a specific
-device group.  An rollout is not instant, but rather each device in a given group will update to
-this wave's targets at some point in time which is determined by many factors: most important being
-network conditions between a device and update servers, as well as a device update schedule.`,
+	rollout := &cobra.Command{
+		Use:   "rollout <wave> -g <group>",
+		Short: "Roll out a wave to a subset of devices",
+		Long: `Roll out a wave to a subset of devices in a factory (in a device group) matching a wave's tag.
+Upon rollout a wave becomes available as an update source for production devices (in a given device group).
```

Comment:
```suggestion
Upon rollout a wave becomes available as an update source for production devices in a given device group.
```
Totally optional, but the parenthesis can be dropped.

Reply:
Parenthesis there is an emphasis that the device group constraint is optional (only applies when the user passed the group argument).

But, as we drop `(in a device group)` above, I think I can paraphrase this to be _Upon rollout a wave becomes available as an update source for **a given subset** of production devices._